3p/vendors/myua.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Copyright notices are no longer required. Can you remove them from all files in this PR? Thanks!

Reply:
Sure. All cleared

Reply:
Sure. All cleared